.circleci/config.yml

```diff
@@ -401,7 +392,7 @@ workflows:
           requires:
             - hold
           <<: *release-branches
-      - deploy-ios:
+      - push-pods:
```

Comment:
Yes, I just renamed the job.

> do we need to install-cocoapods in that job?

I'm pretty sure we don't, since that's not executed currently in the current job (each job is executed in its own container so it shouldn't be shared afaik). Lmk if you think otherwise.

Reply:
Right. Makes sense!